tests/sentry/integrations/github_enterprise/test_client.py

```diff
@@ -147,9 +141,7 @@ def test_get_codeowner_file(self, mock_check_file):
         result = self.install.get_codeowner_file(
             self.config.repository, ref=self.config.default_branch
         )
-
         assert (
-            responses.calls[3].request.headers["Content-Type"] == "application/raw; charset=utf-8"
+            responses.calls[1].request.headers["Content-Type"] == "application/raw; charset=utf-8"
         )
-        assert responses.calls[3].request.headers["Accept"] == "application/vnd.github.raw"
```

Comment:
We no longer need to test for this `Accept` header, since it's tested on the proxy client test (`GithubProxyClientTest.test_authorize_request_valid`). When authored for the region, these headers are missing (as expected)